--- status.proto	2019-07-18 22:38:07.439098015 +0000
+++ src/main/protobuf/google/rpc/status.proto	2019-07-18 22:39:02.664708814 +0000
@@ -17,6 +17,11 @@
 package google.rpc;
 
 import "google/protobuf/any.proto";
+import "scalapb/scalapb.proto";
+
+option (scalapb.options) = {
+  package_name: "com.google.rpc.scala"
+};
 
 option go_package = "google.golang.org/genproto/googleapis/rpc/status;status";
 option java_multiple_files = true;
